bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tWZj_NLO_5f/tWZj_NLO_5f_proc_card.dat

```diff
@@ -0,0 +1,42 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.6.4                 2018-11-09         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+# Model restricted to massless b-quarks
+import model Yt_UFO-nobmass
+# Re-define proton and jet to remove b-quark contribution from proton/jet
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p b > j w- z t QCD=0 QED=99 QNP=2
+add process p b~ > j w+ z t~ QCD=0 QED=99 QNP=2
```

Comment:
Hi, more importantly - this will give you asymmetric events.
You need to add the counter part "b p"

Reply:
1. Correct
2. I guess QNP is some old nomenclature? I've seen it used in various theorists cards. Is it problematic?

Reply:
i didn't notice that, i tested it using MG331, it may be correct in older version. it's fine if you already tested it.

Reply:
> Hi, more importantly - this will give you asymmetric events. You need to add the counter part "b p"

What do you mean by asymmetric? I think these describe the same contribution to the amplitude and considering both would be double counting. To check this, a while back I'd checked for changes in the diagrams generated for e.g. the " u b > u w- z t " process when generating " p p > j w- z t ". Although both the "u b > " and "b u > " subprocesses are there, the number of diagrams are the same, so I think this means Madgraph indeed considers them degenrate.

Reply:
hi, did you check the xs, if you do such test, define p = u u~, and generate three processes:
1. p u > e+ e-, xs=179pb
2. u p > e+ e-, xs=179pb
3. p p > e+ e-, xs=359pb

Reply:
You're right:
pb>tWZj
Cross-section :   0.003717 +- 3.288e-05 pb
pb>tWZj + bp>tWZj :
Cross-section :   0.007483 +- 4.984e-05 pb

I will add this to the card, thank you!

Reply:
The issue with adding these to processes is that the madspin step then takes a lot of memory. I tested running such a card in MG. I first asked Madspin to do leptonic decays only which worked fine. I then tried running for all Madspin decays while monitoring the memory. The program incrementally increased to 11G Virt 10.3G Res (last my memory monitoring script checked) before returning "Exiting due to channel error." Do you have any suggestions to reduce the memory required in the Madspin step? Otherwise, I suppose one solution is to run seperate cards for the top and anti-top diagrams like for single top processes.